extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -771,22 +759,22 @@ private function doStats( $days = null, $lang = null ) {
 
 		$dbr = WikiFactory::db( DB_SLAVE );
 		$res = $dbr->select(
-			array( "city_list" ),
-			array(
+			[  "city_list"  ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "city_list" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)